### ISSUE TITLE
EXE property sheet is empty except for the headers in some files

### ISSUE THREAD

**Issue by Masamune3210** [GerbilSoft/rom-properties#257]
This is with the 1.7 release, though the issue could have been earlier as I don't regularly check the properties of exe files
Can check older versions if needed
This is the exe for rclone 1.53.1
![image](https://user-images.githubusercontent.com/1053504/93727987-77f42d00-fb83-11ea-86ef-e3df7157687b.png)

**Comment by Masamune3210** [comment]
It seems to actually be a issue with property sheets in general, as its happening with other file types
![image](https://user-images.githubusercontent.com/1053504/93728317-09b06a00-fb85-11ea-8183-9b360e59f23f.png)

**Comment by GerbilSoft** [comment]
Already found out about this. It's a last-minute change I did in some code that reduce the width of the description fields. Originally I used an `std::vector<>`, which zero-initializes the values, but then I changed it to `unique_ptr<int[]>` to save a bit of memory. Unfortnuately, `new int[]` doesn't zero-initialize the array. (Strangely, it seemed to work fine on Windows 7...)

Solution: `new int[]()`, which default-initializes the values. In the case of `int`, default is zero.

I'll be releasing a v1.7.1 in a few minutes. This only affects the Windows version.
